metadata-ingestion/src/datahub/ingestion/source/aws/sagemaker_processors/jobs.py

```diff
@@ -767,7 +767,7 @@ def process_processing_job(self, job: Dict[str, Any]) -> SageMakerJob:
 
         input_datasets = {}
 
-        inputs = job["ProcessingInputs"]
+        inputs = job.get("ProcessingInputs", [])
```

Comment:
Classic example of us assuming things about foreign API objects! Note this one in our books.